staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -135,10 +135,33 @@ type ExposeServiceOptions struct {
 	genericiooptions.IOStreams
 }
 
-// NewExposeServiceOptions creates a new ExposeServiceOptions and return a pointer to the
-// struct
-func NewExposeServiceOptions(ioStreams genericiooptions.IOStreams) *ExposeServiceOptions {
-	return &ExposeServiceOptions{
+// exposeServiceFlags is a struct that contains the user input flags to the command.
+type ExposeServiceFlags struct {
+	cmdutil.OverrideOptions
```

Comment:
Why do we have this?

Reply:
It's been there before, see https://github.com/kubernetes/kubernetes/blob/a8cc22f416c9f2d8f9a1a3e31625ac9bc19c162b/staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go#L92

Even though it doesn't do anything I think it'd be nice to review this and run, I see both of these are using those `cmdutil.OverrideOptions` and get rid of that, iow. start the deprecation process and we'll drop those in 1.30.